www/js/control/ProfileSettings.jsx

```diff
@@ -215,42 +472,46 @@ const ProfileSettings = () => {
 
     return (
         <>
-           <SettingRow textKey="control.profile" iconName='logout' action={logOut} desc={settings?.auth?.opcode} descStyle={styles.monoDesc}></SettingRow>
-           <DemographicsSettingRow></DemographicsSettingRow>
-           <SettingRow textKey='control.view-privacy' iconName='eye' action={viewPrivacyPolicy}></SettingRow>
-           <SettingRow textKey="control.view-qrc" iconName="grid" action={viewQRCode}></SettingRow>
-           {timePicker}
-           <SettingRow textKey="control.tracking" action={userStartStopTracking} switchValue={collectSettings.trackingOn}></SettingRow>
-           <SettingRow textKey="control.app-status" iconName="check" action={fixAppStatus}></SettingRow>
-           <SettingRow textKey="control.medium-accuracy" action={toggleLowAccuracy} switchValue={collectSettings.lowAccuracy}></SettingRow>
-           <SettingRow textKey={carbonDatasetString} iconName="database-cog" action={() => setCarbonDataVis(true)}></SettingRow>
-           <SettingRow textKey="control.force-sync" iconName="sync" action={forceSync}></SettingRow>
-           <SettingRow textKey="control.share" iconName="share" action={share}></SettingRow>
-           <SettingRow textKey="control.download-json-dump" iconName="calendar" action={openDatePicker}></SettingRow>
-           {logUploadSection}
-           <SettingRow textKey="control.email-log" iconName="email" action={emailLog}></SettingRow>
-
-            {userDataSection}
+        <Appbar.Header statusBarHeight={12} elevated={true} style={{ height: 46, backgroundColor: 'white', elevation: 3 }}>
+            <Appbar.Content title={t("control.profile")} />
+            <Text>{t('control.log-out')}</Text>
+            <IconButton icon="logout" onPress={() => setLogoutVis(true)}></IconButton>
+        </Appbar.Header>
```

Comment:
addition of `Appbar` is what allowed the logout button to go all the way to the top of the screen, and matches the label screen, characteristic of UI making progress into React